src/gridstack.ts

```diff
@@ -828,8 +837,7 @@ export class GridStack {
    * grid.el.addEventListener('added', function(event) { log('added ', event.detail)} );
    *
    */
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public on(name: GridStackEvent, callback: (event: Event, arg2?: GridItemHTMLElement | GridStackNode[]) => void): GridStack {
+  public on(name: GridStackEvent, callback: GridStackEventHandlerCallback): GridStack {

```

Comment:
not correct because you changed arg2 from GridItemHTMLElement | GridStackNode[] to only GridItemHTMLElement 
we use the array of nodes in change and other callbacks.

Reply:
Yep, you are correct. Apologies I just started working with this library and trying to help contribute a small fix. `arg2` can also be GridStackNode for the dropped event listener and also we need to account for a third `helper` param of type `GridStackNode`